sample/ApplicationInsightsJavaScriptSnippetSample/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNetCore.ApplicationInsights.HostingStartup;
+using Microsoft.AspNetCore.Hosting;
+
+namespace ApplicationInsightsJavaScriptSnippetSample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = new WebHostBuilder()
+                .UseKestrel()
+                .UseContentRoot(Directory.GetCurrentDirectory())
+                .UseStartup<Startup>();
+            new ApplicationInsightsHostingStartup().Configure(builder);
```

Comment:
Use the ASPNETCORE_HOSTINGSTARTUP environment variable instead. 
https://github.com/aspnet/AzureIntegration/blob/4c8b548b309a47f545367501a3e3d73b9dc7cf85/sample/ApplicationInsightsHostingStartupSample/Properties/launchSettings.json#L16